clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -108,16 +109,16 @@ public boolean isConnecting(String id) {
     }
 
     /**
-     * Enter the connecting state for the given connection.
+     * Enter the connecting state for the given connection, moving to a new resolved address if necessary.
      * @param id the id of the connection
-     * @param now the current time
-     * @throws UnknownHostException 
+     * @param now the current time in ms
      */
```

Comment:
Can we add JavaDoc for `host` and `clientDnsLookup`, while we're changing this?

Reply:
Done